content/hardware/04.pro/boards/portenta-x8/tutorials/13.wordpress-webserver/content.md

```diff
@@ -64,39 +66,65 @@ services:
+First, we create a directory where we want to add our **docker-compose.yml** file. Using the `mkdir` command we will create a directory named "wordpress-test". Navigate into this directory with a simple `cd` command. Either copy the docker-compose.yml file into this directory or create it directly here.
+
+To create the file, we can use `cat > docker-compose.yml`, this will create the file, so you can copy the content of the file from above and paste it. Push enter once to go to a new line and press `ctrl C` to exit the file editor. To copy the file from your computer onto the device use:
+
+```
+adb push <path to docker-compose.yml file> /home/fio/wordpress-test
+```
+
+Alternatively, you could place the `docker-compose.yml` file inside the `wordpress-test` directory and push the file using the following command:
+
+```
+adb push .\wordpress-test\ /home/fio
+```
+
+Both options work fine and depend on how you would like to handle the file.
 
 ![cd into correct directory](assets/webserver-mkdir.png)
 
-Before installing the containers, make sure that no other container is running on the ports that the WordPress container will use. You can check what containers are running and what port they are using by running the `docker ps -a` command. This will show a list of the currently installed and running containers on the Portenta X8. To remove a container first stop it with `docker stop <container id>`, then you can run `docker rm <container id>` to remove it. If you want more information about handling containers on your Portenta X8, take a look at our [managing containers with docker tutorial](https://docs.arduino.cc/tutorials/portenta-x8/docker-container).
+***Remember that you may need to run the next command to gain admin access for running the Docker's commands: `sudo su -`  which default password is `fio`***
+
+Before installing the containers, make sure that no other container is running on the ports that the WordPress container will use. You can check what containers are running and what port they are using by running the `docker ps -a` command. This will show a list of the currently installed and running containers on the Portenta X8.
+
+To remove a container first stop it with `docker stop <container id>`, then you can run `docker rm <container id>` to remove it. If you want more information about handling containers on your Portenta X8, take a look at our [Managing Containers with Docker tutorial](https://docs.arduino.cc/tutorials/portenta-x8/docker-container).
```

Comment:
I have separated the paragraph a bit for better readability since it has a lot of commands that users need to follow. How do you see this change? Please feel free to revert the change if you would like to.